docs/learning/example-tools.md

```diff
@@ -0,0 +1,90 @@
+
+![stripper](https://github.com/kozikow/gaggiuino.github.io/assets/722866/1950fd54-5f0d-4bfd-86a4-880c2beb114d)
+
+However, for kits that include low-voltage work, such as tiny JST cables, an automated stripper might not be suitable as it could pull the cables apart. If stripping ever results in any metal strands coming off with the insulation, trim the end and strip again. For low-voltage cable stripping, gently heating the insulation and then pulling it off with pliers or even fingers, without cutting, can be effective. This method, as demonstrated in [this YouTube video](https://www.youtube.com/watch?v=n5o2drU65UM), can be used.
+
+## Solder - Optional
+
+Soldering in your Gaggiuino project can either be a necessity or an option, depending on the approach you choose.
+
+For those assembling without a pre-made kit, soldering is essential. Additionally, some kits may not include pre-assembled low-voltage cabling, making soldering a recommended step for securing low-voltage connections. While handling the cables, avoid pulling them, but tight space inside Gaggia machine may make it hard and soldered connections will be much more secure.
+
+Soldering cables together is simpler and more cost-effective than many assume, particularly when only soldering wires. For guidance, this instructional video offers excellent tips on cable soldering: [How to Connect Cables with a Soldering Iron](https://www.youtube.com/watch?v=Zu3TYBs65FM). Additionally, consult the [Solder Recommendations](learning-sources.md) for best practices and always employ the "Western Union" splice technique. "Splicing cables" refers to connecting two cables together.
+
+When it comes to soldering low-voltage wires, even the most affordable soldering iron is sufficient. Ensure good ventilation or use a fan to avoid inhaling harmful fumes. The soldering process involves melting the solder (the wire in the middle above) by touching it with a hot iron (on the left), which then fuses the cables together more robustly.
+
+![cheap_low_voltage](https://github.com/kozikow/gaggiuino.github.io/assets/722866/0631dcd4-7a85-4fe4-90af-71caa739cdf6)
+
+## Multimeter - Optional
+
+While not mandatory, a multimeter proves to be extremely useful. Should any component fail to operate correctly, conducting multimeter tests is the recommended initial troubleshooting step. Ideally, all components should undergo multimeter testing prior to the initial power-up. Before turning on the device for the first time, verify the continuity of both high-voltage (HV) and low-voltage (LV) lines to ensure there are no short circuits where they shouldn't exist, and to confirm continuity where necessary (e.g., between pins on various components). Ignoring these precautionary tests and directly powering on the machine could potentially harm your board, necessitating a replacement. Below is an example demonstrating how to identify a failed SSR connection using a multimeter:
```

Comment:
> Ignoring these precautionary tests and directly powering on the machine could potentially harm your board, necessitating a replacement

Maybe this warning should be stronger? As an example, someone could have a short circuit of the HV lines through the metal case of the Gaggia,  which will eventually trip a breaker in their house, but in the meantime the case would be live. Depending on if what or who is touching it at the time, that would be a bad day.

Reply:
Thank you. Rewrote this section:

While a multimeter is not strictly required, it is immensely beneficial. Conducting multimeter tests is the first recommended step for troubleshooting should any component fail to function as expected. It's ideal for all components to be tested with a multimeter before the initial power-up. Before activating the device for the first time, it's crucial to check the continuity of both high-voltage (HV) and low-voltage (LV) lines to prevent short circuits where they shouldn't be and ensure continuity where it's needed (e.g., between pins on different components). Neglecting these preventive measures and powering on the device without prior testing could not only damage your board, necessitating a costly replacement but also pose a serious safety risk.

Important Safety Warning: Bypassing these preliminary tests and immediately turning on the machine can lead to severe consequences beyond just damaging the board. For instance, a short circuit in the HV lines through the metal case of the Gaggia machine could lead to the case becoming electrified. This situation could quickly escalate, potentially tripping a breaker in your home. However, the more immediate danger is the risk of electric shock to anyone or anything that comes into contact with the live case, leading to a perilous situation. Always prioritize safety by conducting thorough pre-use checks to mitigate any risks of injury or damage.

Below is an example of multimeter test to detect a failed SSR connection: